ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -972,6 +1028,66 @@ public void shouldGetServerInfo() throws Exception {
     assertThat(serverInfo.getKafkaClusterId(), is(expectedClusterId));
   }
 
+  @Test
+  public void shouldListConnectors() throws Exception {
+    // Given:
+    makeKsqlRequest("CREATE SOURCE CONNECTOR " + TEST_CONNECTOR + " WITH ('connector.class'='" + MOCK_SOURCE_CLASS + "');");
```

Comment:
If all three of these tests create the same connector, why do whichever tests run second and third not fail (because the connector already exists)?

Reply:
The statement fails but doesn't throw an error. I ended up moving this to setUp and tearDown though.